app/Models/Bom.php

```
@@ -0,0 +1,50 @@
+        'unit',
+        'unit_price',
+        'amount',
+        'additional_details',
+        'source_type',
+    ];
+
+    protected static function booted()
+    {
+        static::saving(function ($bom) {
+            $quantity   = $bom->quantity ?? 0;
+            $unit_price = $bom->unit_price ?? 0;
```

PHPMD (2.15.0)

The variable $unit_price is not named in camelCase. (undefined)

(CamelCaseVariableName)